lib/bounces.js

```diff
@@ -120,14 +120,7 @@ module.exports = function (log, error) {
 
           // Flow events have a specific event and structure that must be emitted.
           // Ref `gather` in https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js
-          const flowEventInfo = {
-            event: eventName,
-            time: currentTime,
-            flow_id: flowId,
-            flow_time: elapsedTime
-          }
-
-          log.flowEvent(flowEventInfo)
+          log.info({ op: 'log.flowEvent', event: eventName, time: currentTime, flow_id: flowId, flow_time: elapsedTime })
```

Comment:
Fwiw, `op` here should just be `flowEvent` if we want to match what it is everywhere else (I assume we do). But yeah, let's try not to use `log.info` at all, if we can.